container/container.go

```diff
@@ -227,3 +234,51 @@ func (a *AvroContainerWriter) AddAvroContainerWriter(p *generator.Package) {
 	p.AddFunction(a.filename(), a.name(), "WriteRecord", a.writeRecordDef())
 	p.AddFunction(a.filename(), a.name(), "Flush", a.flushDef())
 }
+
+func (a *AvroContainerWriter) AddCheckSchema(p *generator.Package) {
```

Comment:
What does project start time mean? When the consumer starts? It's a long-running process. What happens if we change the schema between when the consumer starts and the next time it's bounced (which is hopefully never)